### ISSUE TITLE
Constraints stop working.

### ISSUE THREAD

**Issue by z26** [xibyte/jsketcher#99]
Hi,

I don't know how this happens exactly, but at some point while editing a sketch in the dedicated sketch view, the constraints stopped working (as in they still exist, but they are ignored when moving sketch points around with the mouse)

I'll edit this post with more info if I replicate this issue, intentionally or accidentally.

EDIT: this definitively seems to be caused by improperly using a named dimension. An exception must be aborting the solver code if there is any unrecognized constraint. Ideally, only the problematic constraint should be disabled, instead of the whole solver.

**Comment by xibyte** [comment]
Are you using a build from the latest master branch? If so it is currently
broken in the master branch and I am fixing it.

**Comment by z26** [comment]
I'm going to the the web-cad.org website by typing the url.

**Comment by mmiscool** [comment]
The web-cad.org is periodically updated. There is a bunch of UI stuff and bug fixes happening now so it will be updated after the master branch is in a good state.

**Comment by mmiscool** [comment]
@z26 
This is a very old issue. Could you confirm this is still a problem or not? Seeing if we can close this one.

**Comment by z26** [comment]
It's been quite long so I don't remember the exact steps I took back then, but i am unable to recreate this issue.

While investigating this, I did find that setting a distance constraint to A then writing something wrong on purpose like "A = x" (if x doesnt exist) will cause the app to become somewhat but not fully unresponsive. Not a serious problem (the situation is recoverable by deleting the offending statement) but might as well mention it regardless.
